plugins/obs-ffmpeg/obs-ffmpeg-source.c

```diff
@@ -244,7 +250,8 @@ static void ffmpeg_source_open(struct ffmpeg_source *s)
 				s->input, s->input_format,
 				s->buffering_mb * 1024 * 1024,
 				s, get_frame, get_audio, media_stopped,
-				preload_frame, s->is_hw_decoding, s->range);
+				preload_frame, s->is_hw_decoding, s->is_not_seekable,
```

Comment:
Style:  This isn't properly wrapped at 80 columns.  Tabs are 8 columns, so this and the below line should be written like this:

```
				preload_frame, s->is_hw_decoding,
				s->is_not_seekable, s->range);
```

Each line is indented with four (4) tabs.